### PR TITLE
Spike opentonic green

### PR DESCRIPTION
**Pull request by enxebre**

### CODE REVIEW

File: modules/aws/vpc/sg-master.tf [hunk 1]
```diff
@@ -65,7 +65,7 @@ resource "aws_security_group_rule" "master_ingress_https" {
   protocol    = "tcp"
   cidr_blocks = ["0.0.0.0/0"]
   from_port   = 6443
-  to_port     = 6443
+  to_port     = 6445
```
Comment (squat) [inline modules/aws/vpc/sg-master.tf]: what's running on 6444 and 6445?

File: modules/bootkube/resources/bootkube.sh [hunk 2]
```diff
@@ -58,9 +58,10 @@ export ETCDCTL_API=
 
 # shellcheck disable=SC2154
 /usr/bin/docker kill $${signer_id}
-
 rm /etc/kubernetes/manifests/tectonic-node-controller-pod.yaml
 
+cp -r $(pwd)/bootstrap-configs /etc/kubernetes/bootstrap-configs
```
Comment (squat) [inline modules/bootkube/resources/bootkube.sh]: why not just `cp -r bootstrap-configs /etc/kubernetes/bootstrap-conf` without the `$(pwd)`?

File: modules/bootkube/resources/manifests/03-openshift-web-console-namespace.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  # This is the namespace used to hold the openshift console.
+  # They require openshift console run in this namespace.
```
Comment (squat) [inline modules/bootkube/resources/manifests/03-openshift-web-console-namespace.yaml]: Who is `they`?

File: modules/ignition/resources/kubernetes/runtime-mappings.yaml [hunk 4]
```diff
@@ -9,3 +9,5 @@ versions:
         docker: [ "17.03", "1.12"]
     1.9:
         docker: [ "17.03", "1.12"]
+    3.10:
+        docker: [ "17.03", "1.12"]
```
Comment (squat) [inline modules/ignition/resources/kubernetes/runtime-mappings.yaml]: we should keep the newline

File: modules/ignition/resources/services/kubelet.service [hunk 5]
```diff
@@ -2,44 +2,48 @@
...
   --volume=resolv,kind=host,source=/etc/resolv.conf \
   --mount volume=resolv,target=/etc/resolv.conf \
   --volume var-lib-cni,kind=host,source=/var/lib/cni \
   --mount volume=var-lib-cni,target=/var/lib/cni \
   --volume var-lib-kubelet,kind=host,source=/var/lib/kubelet \
   --mount volume=var-lib-kubelet,target=/var/lib/kubelet \
   --volume var-log,kind=host,source=/var/log \
-  --mount volume=var-log,target=/var/log"
+  --mount volume=var-log,target=/var/log \
+  --insecure-options=image"
```
Comment (squat) [inline modules/ignition/resources/services/kubelet.service]: Why do we need to use `--insecure-options`? Can we instead add a task to make the image secure?

File: modules/tectonic/resources/manifests/secrets/ingress-tls.yaml [hunk 6]
```diff
@@ -1,9 +1,10 @@
...
-  namespace: tectonic-system
+  name: tectonic-ingress-tls
+  namespace: tectonic-ingress
 type: Opaque
 data:
   tls.crt: ${ingress_tls_cert}
-  tls.key: ${ingress_tls_key}
\ No newline at end of file
+  tls.key: ${ingress_tls_key}
+  bundle.crt: ${ingress_tls_bundle}
```
Comment (squat) [inline modules/tectonic/resources/manifests/secrets/ingress-tls.yaml]: I see that this commit didn't introduce the mistake but we should add a newline here.

File: modules/ignition/resources/services/kubelet.service (hunk 5 above)
Comment (squeed) [reply modules/ignition/resources/services/kubelet.service]: because we're pulling a docker image instead of an ACI. No choice :-(

Comment (squeed) [reply modules/ignition/resources/services/kubelet.service]: This goes away when we switch to quay.

Comment (squat) [reply modules/ignition/resources/services/kubelet.service]: Right, this is my point. Can we either:
a: switch to quay right now
or
b: make a task to switch to quay and remove this flag

**Comment by squeed** [comment]
I did a quick smoketest of this on libvirt, and it's good enough for now. There are some followup items:

1. tectonic-torcx (k8s-node-bootstrap.service) isn't working and should probably be deleted
2. I accidentally ran out of disk space on my testing VM and the openshift-apiserver was evicted. It needs the critical tag.

But this looks good! LGTM.

**Pull request merged**